go.mod

```diff
@@ -1,29 +1,237 @@
 module github.com/edoardottt/cariddi
 
-go 1.18
+go 1.22.0
+
+toolchain go1.22.2
 
 require (
 	github.com/fatih/color v1.17.0
 	github.com/gocolly/colly v1.2.0
+	github.com/trufflesecurity/trufflehog/v3 v3.73.0
```

Comment:
I'm leaving this comment here but it's not related to this line specifically.

The goreleaser CI is giving these errors:
```
Error: ../../../go/pkg/mod/github.com/trufflesecurity/trufflehog/v3@v3.73.0/pkg/gitparse/gitparse.go:200:18: cannot use defaultMaxDiffSize (untyped int constant 2147483648) as int value in struct literal (overflows)
Error: ../../../go/pkg/mod/github.com/trufflesecurity/trufflehog/v3@v3.73.0/pkg/gitparse/gitparse.go:201:18: cannot use defaultMaxCommitSize (untyped int constant 2147[48](https://github.com/edoardottt/cariddi/actions/runs/9128423415/job/25100742967?pr=150#step:4:49)3648) as int value in struct literal (overflows)
```

Do you have any clue about what they mean?